packages/core/webpack.config.dist.js

```diff
@@ -3,7 +3,7 @@ var webpack = require("webpack");
 const MonacoWebpackPlugin = require('monaco-editor-webpack-plugin');
 
 module.exports = {
-  mode: "production",
+  mode: "development",
```

Comment:
Did you mean to change this to "development"?

Reply:
To debug, yeah. That should be prod.

Are these webpack configs used for just the playground?